@xen-orchestra/lite/src/libs/xen-api.ts

```diff
@@ -278,9 +280,66 @@ export default class XenApi {
 
   async injectWatchEvent(poolRef: string) {
     this.#fromToken = await this.#call("event.inject", [
-      this.sessionId,
+      this.getSessionId(),
       "pool",
       poolRef,
     ]);
   }
+
+  get vm() {
+    return {
+      start: ({ vmsRef }: { vmsRef: string[] }) =>
```

Comment:
Should we also accept a singular `vmRef` and fallback to it: `({ vmRef, vmRefs = [vmRef] }) =>`. Or do you think it's not neccesary?